server/src/main/java/org/opensearch/extensions/ExtensionsManager.java

```diff
@@ -223,14 +222,16 @@ public ExtensionActionResponse handleTransportRequest(ExtensionActionRequest req
         return extensionTransportActionsHandler.sendTransportRequestToExtension(request);
     }
 
-    private void registerRequestHandler() {
+    private void registerRequestHandler(ActionModule actionModule) {
```

Comment:
Could we also pass `DynamicActionRegistry` here? The `ActionModule` is not really needed

Reply:
Done